libyul/optimiser/Suite.cpp

```diff
@@ -383,6 +383,25 @@ void OptimiserSuite::validateSequence(std::string_view _stepAbbreviations)
 	assertThrow(nestingLevel == 0, OptimizerException, "Unbalanced brackets");
 }
 
+bool OptimiserSuite::isEmptyOptimizerSequence(std::string const& _sequence)
```

Comment:
```suggestion
bool OptimiserSuite::isEmptyOptimiserSequence(std::string const& _sequence)
```

In multiple places around the code `optimiser` is already written with `s` instead of `z`, so maybe we should stick with one form?

Reply:
This is an age old issue - in general, we want to refer to it as `optimizer`, and refactor all of the `optimiser` occurrences, but this is for another PR.

Reply:
We should have added `OptimiserSuite::isEmptyOptimizerSequence` to ``libyul/optimiser/OptimizerUtilities.h`` :-).

Reply:
> We should have added `OptimiserSuite::isEmptyOptimizerSequence` to `libyul/optimiser/OptimizerUtilities.h` :-).

Neat, I can do that, should be a small change.

Reply:
Wait, damn, you actually did that :-)? I was just joking because of the ``s`` and ``z`` even of the path of that file :-D

Reply:
Maybe I should have been clearer that that wasn't meant seriously, sorry :-). But it makes little sense to *actually* move it there, I just found the symmetrie between having ``s`` in class name and ``z`` in function name in the member function here and ``s`` in the direction name and ``z`` in the file name in that header funny :-).

Reply:
Didn't realize you were joking since the `s`/`z` is mixed all over the place anyway, and I did ask Kamil last week what would be a good place to put this function in (StringUtils, etc.). In any case, it was in a separate commits, so it was just a matter of reverting.